src/main/java/org/broadinstitute/hellbender/utils/recalibration/RecalibrationReport.java

```diff
@@ -325,12 +325,33 @@ else if (argument.equals("binary_tag_name"))
                 RAC.BINARY_TAG_NAME = (value == null) ? null : (String) value;
 
             else if (argument.equals("sort_by_all_columns"))
-                RAC.SORT_BY_ALL_COLUMNS = Boolean.parseBoolean((String) value);
+                RAC.SORT_BY_ALL_COLUMNS = decodeBoolean(value);
         }
 
         return RAC;
     }
 
+    private byte decodeByte(final Object value) {
+        if (!(value instanceof Byte || value instanceof String)){
+            throw new IllegalArgumentException("expected a Byte or a String but got " + value);
+        }
+        return value instanceof Byte ? (Byte)value: Byte.parseByte((String)value);
```

Comment:
Cache the first `instanceof` check in these `decode*` methods, as `instanceof` is actually a bit expensive.